docs/ui/components/Collapsible/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { css } from '@emotion/react';
+import { borderRadius, iconSize, shadows, spacing, theme } from '@expo/styleguide';
+import React, { PropsWithChildren, ReactNode } from 'react';
+
+import { HEADLINE } from '~/ui/components/Text';
+import { durations } from '~/ui/foundations/durations';
+import { ChevronDownIcon } from '~/ui/foundations/icons';
+
+type CollapsibleProps = PropsWithChildren<{
+  /** The content of the collapsible summary */
+  summary: ReactNode;
+  /** If the collapsible should be rendered "open" by default */
+  open?: boolean;
+  testID?: string;
+}>;
+
+export function Collapsible({ summary, open, testID, children }: CollapsibleProps) {
+  return (
+    <details css={detailsStyle} open={open} data-testid={testID}>
+      <summary css={summaryStyle}>
+        <ChevronDownIcon css={markerStyle} size={iconSize.small} />
+        <HEADLINE tag="span" css={headlineStyle}>
+          {summary}
+        </HEADLINE>
+      </summary>
+      <div css={contentStyle}>{children}</div>
+    </details>
+  );
+}
+
+const detailsStyle = css({
+  overflow: 'hidden',
+  background: theme.background.default,
+  border: `1px solid ${theme.border.default}`,
+  borderRadius: borderRadius.medium,
+  padding: 0,
+
+  '&[open]': {
+    boxShadow: shadows.tiny,
+  },
+});
+
+const summaryStyle = css({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  userSelect: 'none',
+  backgroundColor: theme.background.secondary,
+  padding: spacing[1.5],
+  paddingRight: spacing[3],
+  margin: 0,
+
+  '&::marker': { content: '""' },
+});
+
+// TODO(cedric): remove this when we removed all `h4` tags in the MDX files
+const headlineStyle = css({
```

Comment:
This at least makes the h4 inside summaries a bit better. I recon we want to merge this before editing all content, so thought I put this here.